pkg/gh/github.go

```diff
@@ -86,6 +63,41 @@ func (o GitOptions) handleRateLimit(action func() (*github.Response, error)) err
 	return nil
 }
 
+func (o GitOptions) handleRateLimitList(action func(opt *github.ListOptions) (*github.Response, error)) error {
+	opt := &github.ListOptions{
+		PerPage: 30,
+	}
+
+	for {
+		resp, err := action(opt)
+
+		if resp.StatusCode == http.StatusUnauthorized {
+			return errors.Wrap(err, "failed to auth with GitHub, does your personal access token have the repo scope? https://github.com/settings/tokens/new?scopes=repo")
+		}
+
+		if err != nil {
+			if githubErr := github.CheckResponse(resp.Response); githubErr != nil {
```

Comment:
Maybe worth to create a new separate func for this one?

Reply:
thanks for the feedback - updated